### PR TITLE
Added login form

### PR DESCRIPTION
**Pull request by max1k1**
good form!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import './App.css';
+import "./App.css";
+import LoginForm from "./components/forms/LoginForm/LoginForm";
 
 function App() {
   return (
     <div className="App">
```
Comment (Io38) [inline src/App.js:6]: what do you need className "App" for

File: src/components/Input/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "./Input.module.css";
+
+const Input = (props) => {
```
Comment (Io38) [inline src/components/Input/Input.jsx:4]: you do not destructure props object. It leads to unreadable code

File: src/components/forms/LoginForm/LoginForm.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.button {
+  margin-top: 20px;
+  margin-bottom: 5px;
+  width: 40%;
+  height: 27px;
+}
+
+.button:hover{
+    transform: translate(10vh,5vw);
+ background-color: red;
```
Comment (Io38) [inline src/components/forms/LoginForm/LoginForm.module.css:27]: install prettier pls!

**Review by Io38 — CHANGES_REQUESTED** [review]
fix comments

File: src/components/forms/LoginForm/LoginForm.module.css (hunk 3 above)
Comment (max1k1) [reply src/components/forms/LoginForm/LoginForm.module.css:27]: ok

**Comment by Io38** [comment]
good job

**Pull request merged**